ts/config.ts

```diff
@@ -45,6 +45,9 @@ const DEFAULT_PAGE_SIZE = 12;
 
 // Default mixpanel EU url
 const DEFAULT_MIXPANEL_URL = "https://api-eu.mixpanel.com";
+// Default sentry dsn url
+const DEFAULT_SENTRY_DSN =
+  "https://43b87dcfc91f9cfdfaf71b254eb8f58e@o4507197393469440.ingest.de.sentry.io/4507221483585616";
```

Comment:
As above.

Reply:
added in [cfd5738](https://github.com/pagopa/io-app/pull/5756/commits/cfd573835d00b98495772751fa91b09a8d2e5dd0)